pkg/types.go

```diff
@@ -1,5 +1,5 @@
 package pkg
 
 const (
-	ManagerAddr = "vizier-core:6789"
+	ManagerAddr = "vizier-core.kubeflow:6789"
```

Comment:
Why is this change added ?

Reply:
kube-dns will assign k8s services a DNS A record for a name of the form my-svc.my-namespace.svc.cluster.local and  my-svc.my-namespace. This resolves to the cluster IP of the Service.
vizier-core is service name, and if we don't add namespace, when Pod in non-kubeflow namespace cannot resolve vizier-core to its cluster IP (it tries to resolve vizier-core.non-kubeflow).

Reply:
Yes. I mean to say that vizier core can run in any namespace